content/en/quickstart/_index.md

```diff
@@ -63,13 +65,9 @@ control plane version: 1.8.2-distro-v0
 data plane version: 1.8.2-distro-v0 (2 proxies)
 ```
 
-Additionally to the version output - a user can confirm that config validation is functioning as expected by issuing the following command:
-
-```sh
-getmesh config-validate
-```
+**Check the Istio Configuration**
 
-With fresh install of Kubernetes cluster and Istio - the output will look similar to the below:
+Run `getmesh config-validate` to confirm that GetMesh can access the Istio configuration. With a fresh install of Kubernetes and Istio, the output will look similar to the below:
```

Comment:
in the docs for `config-validate` we say "Validate the current Istio configurations in your cluster", so it's not just whether GetMesh can access the configruation.

Reply:
ack